go.mod

```diff
@@ -165,6 +166,7 @@ require (
 	github.com/quasilyte/regex/syntax v0.0.0-20210819130434-b3f0c404a727 // indirect
 	github.com/quasilyte/stdinfo v0.0.0-20220114132959-f7386bf02567 // indirect
 	github.com/rivo/uniseg v0.2.0 // indirect
+	github.com/russross/blackfriday/v2 v2.1.0 // indirect
```

Comment:
Are you still using dependencies here?

Reply:
I'm actually not sure.. I didn't add them manually. Maybe they got added by cobra/doc?